src/libpriv/rpmostree-importer.cxx

```diff
@@ -702,8 +704,8 @@ compose_filter_cb (OstreeRepo         *repo,
        * Note that we rewrite opt in handle_translate_pathname, but
        * this gets called with the old path, so handle it here too. */
       if (!(path_is_ostree_compliant (path) ||
-            g_str_equal (path, "opt") ||
-            g_str_has_prefix (path, "opt/")))
+            g_str_equal (path, "/opt") ||
```

Comment:
Context: this is what prompted this PR, as reading through here I was unsure whether `path` was supposed to be an absolute or a relative path.